lib/postrank-uri.rb

```diff
@@ -142,8 +143,24 @@ def unescape(uri)
       str
     end
 
+    def unescape_unreserved(uri)
```

Comment:
My thought had been that `unescape` could be given an individual URI component in which case you actually would want it to unescape all characters. And that it would limit potential side-effects.

In practice though, I haven't found instance where we use `unescape` directly in our code.